aspnetcore-webapi/GettingStartedASPNETMongoDB/GettingStartedASPNETMongoDB/Interfaces/IStudentService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using GettingStartedASPNETMongoDB.Models;
+
+namespace GettingStartedASPNETMongoDB.Interfaces
+{
+    public interface IStudentService
+    {
+        Task<Student> Create(Student student);
+
+        Task Delete(string id);
+
+        Task<List<Student>> GetAll();
+
+        Task<Student> GetById(string id);
```

Comment:
If this can return null when no Course is found, we should mark the return type as nullable with that Task<Student?>